client/src/components/Event/Event.css

```diff
@@ -1,12 +1,47 @@
 .event-component {
-margin: 0;
+    margin: 0;
+    padding: 0;
+    color: white;
+    background-image: url("../Footer/img/star.svg"), url("../Home/AboutUs/Event/mobile-planet.svg");
+    background-size: cover, 25vw;
+    background-position: center, left top -55vh;
+    background-attachment: fixed, local;
+    background-repeat: no-repeat, no-repeat;
+    height: auto;
 }
 
-.event-component .event_title {
+.event-component .event-title {
     font-family: 'Raleway';
     color: #EE2A7B;
     font-size: 8vh;
     opacity: 1;
     margin: 5vh 0;
```

Comment:
here the top margin of `.event-title` is causing that unnecessary margin on the top. it should be `margin-bottom: 5vh ;`

Reply:
I would suggest that this issue can be added to #195 with other minor updates or we can open a separate issue for it with issue of the alignment of columns of the table. and this PR is ready to be merged as here's already a lot of work done.

Reply:
That seems to be working for now. Thanks for pointing out @tusharjain0022! :smile: